code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -327,7 +354,7 @@
 	on = TRUE
 
 /obj/item/clothing/head/helmet/space/hardsuit/syndi/proc/activate_combat_mode()
-	name += " (combat)"
+	name = "[initial(name)] (combat)"
```

Comment:
oh god, how'd this go unnoticed for so long lmao

Reply:
Combat combat combat combat combat combat

Reply:
Also this doesn't fix it, I need to add something else too